### DISCUSSION TITLE
part-ii/2018-134/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#52] [Question comments]
Part II 2018 Riemann Surfaces: Paper 2, Section II, F 

https://questions.tripos.org/part-ii/2018-134/

**Comment by artie2000** [comment]
<details>
<summary>For the example at the end, the easiest option is probably</summary>

$\mathbb{C}$ and $\mathbb{D}$ (Liouville's theorem).
</details>
